src/responder/kcm/kcmsrv_ops.c

```diff
@@ -138,7 +139,8 @@ struct tevent_req *kcm_cmd_send(TALLOC_CTX *mem_ctx,
 
     state->op_ctx->input = sss_iobuf_init_readonly(state->op_ctx,
                                                    input->data,
-                                                   input->length);
+                                                   input->length,
+                                                   true);
```

Comment:
What does `input` contain?

Reply:
My guess is that it could contain a ticket (i.e. when the operation is STORE), so I preferred to secure the data.

We could try to identify which operations contain data to be secured, and set the flag accordingly.

Reply:
Then to be consistent it's required to follow all the way up to the socket read:
kcm_recv() -> kcm_recv_data() + kcm_input_parse() + kcm_cmd_dispatch() -> kcm_cmd_send()

Reply:
If a buffer that contains a copy of `input` needs to be secured, then `input` itself also needs to be secured.